### PR TITLE
Nav Buttons works now

### PR DESCRIPTION
**Pull request by frederik0312**

### CODE REVIEW

File: BDSA2020.View/wwwroot/main.js [hunk 1]
```diff
@@ -38,7 +36,9 @@ document.querySelector('#controls').addEventListener('click', (event) => {
 
 function checkMore(){
     //calculates if there is more desriptions from scroll values (455 is height of one description (px))
-    if((document.getElementById('scroll-container').scrollHeight - 455.0) == document.getElementById('scroll-container').scrollTop){
+    console.log((document.getElementById('scroll-container').scrollLeft - document.getElementById('scroll-container').clientWidth));
+    console.log(document.getElementById('scroll-container').scrollWidth);
```
Comment (bjornarjatten) [inline BDSA2020.View/wwwroot/main.js:40]: Consider removing this

**Review by bjornarjatten — COMMENTED** [review]
Please look at my comments

File: BDSA2020.View/wwwroot/main.js (hunk 1 above)
Comment (simoboye) [inline BDSA2020.View/wwwroot/main.js:40]: Remove these
```suggestion
```

**Review by simoboye — APPROVED** [review]

**Review by bjornarjatten — APPROVED** [review]
Looks good now

**Review by emieboii — APPROVED** [review]
Perfect

**Pull request merged**